setup.py

```diff
@@ -16,15 +16,15 @@
     install_requires=[
         "beautifulsoup4==4.7.1",
         "editdistance>=0.5.2, <0.6.0",
+        "emmental @ git+https://github.com/SenWu/emmental.git@master",
```

Comment:
Ideally, I think we do want a stable versioned released before merging in.

Reply:
Agree! I will release a new version of Emmental soon.